ifrs17/Import/Importers.ipynb

```diff
@@ -1369,13 +1369,20 @@
                 "\n                if(values.Length == 0 && !parsingStorage.MandatoryAocSteps.Contains(new AocStep(aocType, novelty))) return null;",
                 "\n            }",
                 "\n            ",
+                "\n            var accidentYearInput = (datarow.Field<string>(nameof(RawVariable.AccidentYear)));",
```

Comment:
do this only inside the if(hasAccidentYearColumn)

Reply:
Can't move the variables inside the if because the will not be accessible again in line 1385

Reply:
Define accidentYear outside with a default value.
Then do the checks and inside the if.